src/blocks/pomodoro.rs

```diff
@@ -75,21 +84,70 @@ pub struct Config {
     pub blocking_cmd: bool,
 }
 
+enum PomodoroState {
+    Idle,
+    Prompt,
+    Notify,
+    Break,
+    PomodoroRunning,
+    PomodoroPaused,
+}
+
+impl PomodoroState {
+    fn get_block_state(&self) -> State {
+        use PomodoroState::*;
+        match self {
+            Idle | PomodoroPaused => State::Idle,
+            Prompt => State::Warning,
+            Notify => State::Good,
+            Break | PomodoroRunning => State::Info,
+        }
+    }
+
+    fn get_status_icon(&self) -> Option<Cow<'static, str>> {
```

Comment:
Couldn't this be just an `&'static str` instead of cow?

Reply:
Fixed